nemo/collections/common/parts/rnn.py

```diff
@@ -188,10 +204,15 @@ def __init__(
                     bias.data[hidden_size : 2 * hidden_size].fill_(forget_gate_bias)
                 if "bias_hh" in name:
                     bias = getattr(self.lstm, name)
-                    bias.data[hidden_size : 2 * hidden_size].fill_(0)
+                    # bias.data[hidden_size : 2 * hidden_size].fill_(0)
+                    bias.data[hidden_size : 2 * hidden_size] *= float(hidden_hidden_bias_scale)
 
         self.dropout = torch.nn.Dropout(dropout) if dropout else None
 
+        for name, v in self.named_parameters():
```

Comment:
How about to check and if weights_init_scale is 1.0 then skip this part?

Reply:
Its done only once at init, should not cost much time.